eth-benchmark-tests/src/jmh/java/tech/pegasys/teku/benchmarks/ShuffleBenchmark.java

```diff
@@ -60,4 +61,13 @@ public void shuffledListBench(Blackhole bh) {
     miscHelpers.shuffleList(indices, seed);
     bh.consume(indices);
   }
+
+  public static void main(String[] args) {
+    ShuffleBenchmark benchmark = new ShuffleBenchmark();
+
+    benchmark.indexCount = 32768;
+
+    new CustomRunner(2, 2).withBench(benchmark::shuffledIndexBench).run();
+    new CustomRunner(2, 2).withBench(benchmark::shuffledListBench).run();
+  }
```

Comment:
We probably don't need to add this.  The IntelliJ jmh plugin should be able to run the benchmarks directly.

Reply:
I used that to run the profiler. Because it seems not possible to run benchmarks using jmh plugin with the and activate the intellij profiler at the same time.